cg/cli/post_process/utils.py

```diff
@@ -0,0 +1,22 @@
+import re
+
+from cg.models.cg_config import CGConfig
+from cg.services.post_processing.pacbio.post_processing_service import PacBioPostProcessingService
+
+
+def get_post_processing_service_from_run_name(
```

Comment:
This method will have to be expanded when more devices are available to post-process